c03 finished translation by 9TDWorld (#146)
IllusionMods/KKS-Translation: Translation/en/RedirectedResources/assets/abdata/adv/scenario/c03/penetration/penetrator_end/translation.txt

//
// Dumped for Koikatsu Sunshine v1.0.1 (KoikatsuSunshine) by KKS_TextDump v1.4.5.0
//

> 「えへへ、入れてもらえて嬉しかったです。おかげでムラムラもおさまったですよー♪」=“Hehe, I'm so you let me join. Thanks to that my horniness has calmed.ー♪"

Reviewer: ```suggestion
「えへへ、入れてもらえて嬉しかったです。おかげでムラムラもおさまったですよー♪」=“Hehe, I'm so glad you let me join in. Thanks to that my horniness subsided!♪"
```
